Lang/Spanish.pj.Lang

```diff
@@ -234,6 +234,7 @@
 #465#  "Mostrar velocidad"
 #466#  "Velocidad de la pantalla:"
 #467#  "Comprobar si Project64 ya está en ejecución"
+#468#  "Directorio Único de Guardados de Juego"

```

Comment:
ok, but this is used for "save states" or is for native "save games" aka nvram ?